packages/eslint-plugin/tests/docs.test.ts

```diff
@@ -208,10 +208,7 @@ describe('Validating rule metadata', () => {
   }
 });
 
-describe.each([
-  path.join(__dirname, '../README.md'),
-  path.join(__dirname, '../docs/rules/README.md'),
-])('%s', readmePath => {
+describe.each([path.join(__dirname, '../README.md')])('%s', readmePath => {
```

Comment:
> Let me know if you think it's fine to remove the table in the package's README and simply point people to the website.

+1, good call, yes 😄. I'd love to delete all this code and stop having to hand-craft any table!